notebooks/signac_103_A_Basic_Workflow.ipynb

```diff
@@ -340,9 +458,29 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 15,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Computing volume ofComputing volume of 5a6c687f7655319db24de59a2336eff8\n",
```

Comment:
The output of the parallel computation is garbled. Should we skip executing this cell? 🤷

Reply:
I'm not sure that showing off Python's built-in tools for parallel execution even make sense in this notebook. Maybe we just remove those cells.

Reply:
The motivation behind these examples was to show how signac easily integrates with existing workflows and does not necessarily require all of the signac-flow machinery *per se*. This is an important requirement for some people, so I am not sure whether it is a good idea to completely remove it. Might be good to put into context though if kept.

Reply:
I think it makes sense to move this into a separate section, because if someone is looking for this content  they definitely wouldn't think to look here.

Reply:
In #25, I kept the content about process/thread pools but I put it in a Markdown code block instead of making it executable. This makes it easier to have reproducible outputs on a local machine when rendering the notebooks, and it makes the output more sensible (important for new users and readability online).